Content.Shared/_Impstation/CosmicCult/Components/CosmicGlyphTransmuteWeaponComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._Impstation.CosmicCult.Components;
+
+[RegisterComponent]
+public sealed partial class CosmicGlyphTransmuteWeaponComponent : Component
+{
+    /// <summary>
+    ///     The search range for finding transmutation targets.
+    /// </summary>
+    [DataField] public float TransmuteRange = 0.5f;
+
+    /// <summary>
+    ///     A pool of weapons that we pick from when transmuting.
+    /// </summary>
+    [DataField]
+    public List<ProtoId<EntityPrototype>> TransmuteWeapon =
```

Comment:
```suggestion
    public HashSet<ProtoId<EntityPrototype>> TransmuteWeapon =
```
same as above, fixes sandboxing issue, we love RT don't we folks
(just putting these two on github to be 100% clear abt what I meant in discord)